### PR TITLE
Issue 566. Move from deprecated phantom.args to system.args

### PR DESCRIPTION
**Pull request by NameFILIP**
Fixes https://github.com/teampoltergeist/poltergeist/issues/566

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
 #### Bug fixes ####
 
+*   Move from deprecated `phantom.args` to `system.args` (Filip Spiridonov) [Issue 566]
```
Comment (simi) [inline CHANGELOG.md]: I don't think this is bugfix.

**Comment by simi** [comment]
Is this PhantomJS 1.8.x compatible also?

**Comment by NameFILIP** [comment]
Based on the [PhantomJS' changelog](https://github.com/ariya/phantomjs/blob/master/ChangeLog#L214) it is 1.5 compatible

File: CHANGELOG.md (hunk 1 above)
Comment (NameFILIP) [inline CHANGELOG.md]: Moved to features

**Comment by route** [comment]
Thanks guys

**Comment by NameFILIP** [comment]
Super fast, thank you too!

**Pull request merged**